docs/architecture/introduction.rst

```diff
@@ -122,8 +122,6 @@ Notebooks
 learning models, visualize the behavior of trained models, or even manage the training lifecycle of
 a model manually. Determined makes it easy to launch and manage notebooks.
 
-Determined Notebooks have the following benefits:
-
 -  Jupyter Notebooks run in containerized environments on the cluster. We can easily manage
```

Comment:
An alternative approach:

The benefits sentence could be kept, followed by the first two bullet points.

The third and fourth bullet points could go in an admonition (`note::`)

Reply:
yes makes sense!